Resources/Prototypes/_Goobstation/Entities/Objects/Misc/identification_cards.yml

```
@@ -14,4 +77,174 @@
+    key: enum.AgentIDCardUiKey.Key
+    inHandsOnly: true
+  - type: Tag
+    tags:
+    - DoorBumpOpener
+  - type: ChameleonClothing
+    slot: [ idcard ]
+    default: ExecutiveIDCard
+  - type: UserInterface
+    interfaces:
+     enum.AgentIDCardUiKey.Key:
+        type: AgentIDCardBoundUserInterface
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)